### PR TITLE
[CI:DOCS] Fix up example description of podman-container commands

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-container-clone.1.md.in [hunk 1]
```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
...
 
 ## EXAMPLES
+
+Clone current container into a new container:
 ```
 # podman container clone d0cf1f782e2ed67e8c0050ff92df865a039186237a4df24d7acba5b1fa8cc6e7
 6b2c73ff8a1982828c9ae2092954bcd59836a131960f7e05221af9df5939c584
 ```
 
+Clone current container into specified new container:
```
Comment (vrothberg) [inline docs/source/markdown/podman-container-clone.1.md.in:102]:
```suggestion
Clone current container into a new container named "clone":
```

File: docs/source/markdown/podman-container-clone.1.md.in [hunk 2]
```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
 clone process has completed, this specifies a detached running mode.
 
 ## EXAMPLES
+
+Clone current container into a new container:
```
Comment (vrothberg) [inline docs/source/markdown/podman-container-clone.1.md.in:96]: I wondered what "current" means.  Other man pages use "specified" instead which I find clearer and consistent.

File: docs/source/markdown/podman-container-clone.1.md.in (hunk 1 above)
Comment (vrothberg) [reply docs/source/markdown/podman-container-clone.1.md.in]: Still there.

File: docs/source/markdown/podman-container-clone.1.md.in (hunk 2 above)
Comment (vrothberg) [reply docs/source/markdown/podman-container-clone.1.md.in]: Still there

File: docs/source/markdown/podman-container-clone.1.md.in [hunk 3]
```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
...
 
 ## EXAMPLES
+
+Clone specified container into a new container:
 ```
 # podman container clone d0cf1f782e2ed67e8c0050ff92df865a039186237a4df24d7acba5b1fa8cc6e7
 6b2c73ff8a1982828c9ae2092954bcd59836a131960f7e05221af9df5939c584
 ```
 
+Clone specified container into new named container:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-container-clone.1.md.in:102]:
```suggestion
Clone specified container into a newly named container:
```

File: docs/source/markdown/podman-container-clone.1.md.in [hunk 4]
```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
...
 ```
 # podman container clone --name=clone d0cf1f782e2ed67e8c0050ff92df865a039186237a4df24d7acba5b1fa8cc6e7
 6b2c73ff8a1982828c9ae2092954bcd59836a131960f7e05221af9df5939c584
 ```
 
+Replace specified container with selected resource constraints into new container, removing original container:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-container-clone.1.md.in:108]:
```suggestion
Replace specified container with selected resource constraints into a new container, removing original container:
```

File: docs/source/markdown/podman-container-clone.1.md.in [hunk 5]
```diff
@@ -92,21 +92,26 @@ When set to true, this flag runs the newly created container after the
...
 ```
 # podman container clone --destroy --cpus=5 d0cf1f782e2ed67e8c0050ff92df865a039186237a4df24d7acba5b1fa8cc6e7
 6b2c73ff8a1982828c9ae2092954bcd59836a131960f7e05221af9df5939c584
 ```
 
+Clone specified container giving new name and then replacing the image of original container with specified image name:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-container-clone.1.md.in:114]:
```suggestion
Clone specified container giving a new name and then replacing the image of the original container with the specified image name:
```

**Pull request merged**